entrypoint/airgap/airgap_suite_test.go

```diff
@@ -68,11 +70,18 @@ func validateAirgap() {
 		os.Exit(1)
 	}
 
-	if (cfg.Product == "k3s" && strings.Contains(os.Getenv("server_flags"), "protect")) ||
-		(cfg.Product == "rke2" && strings.Contains(os.Getenv("server_flags"), "profile")) {
+	if (cfg.Product == "k3s" && strings.Contains(serverFlags, "protect")) ||
+		(cfg.Product == "rke2" && strings.Contains(serverFlags, "profile")) {
 		shared.LogLevel("error", "airgap with hardened setup is not supported\n")
 		os.Exit(1)
 	}
+
+	if (cfg.Product == "rke2") && (os.Getenv("no_of_windows_worker_nodes") != "0") &&
+		shared.StringInSlice(serverFlags, cniSlice) {
```

Comment:
cniSlice is used only here. How about we set cniSlice to "calico", "flannel" (to the cni's we support) and check StringNotInSlice?

Reply:
This is avoid continuing with  invalid values in server_flags